src/getHumanAge.test.js

```diff
@@ -7,4 +8,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(
+    'should return array with zero values if cat and dog years are less then 15',
+    () => {
+      const result = getHumanAge(10, 10);
+
+      expect(result).toEqual([0, 0]);
+    });
+
+  test(
+    'should return array with 1 values if cat and dog years are less then 24',
```

Comment:
The test description is slightly misleading. It should specify that the function should return [1, 1] when both cat and dog years are exactly 15, not less than 24.